cfpb/sbl-frontend
# Filing app routing - Phase 1: mock data

Part of #331 

## Changes

- fix: Add missing parameter for POST/PUT calls in our shared `request` function
- feat: Create routes for each step of the Filing process
- feat: Derive the status of each step in the Filing process based on the proposed Filing object info
- feat: Create a shared `FilingStepWrapper` to display the step indicator above each page's content

## How to test this PR

1. Visit http://localhost:8899/filing/2024/TESTLEI1234/upload
2. Use `Previous`/`Next` to navigate between pages
3. Confirm that the url corresponds to the `current`ly styled step

## Screenshot
<img width="1677" alt="Screenshot 2024-04-09 at 3 31 28 PM" src="https://github.com/cfpb/sbl-frontend/assets/2592907/bddb998d-2b51-4108-919a-3e265dd96473">
